tests/capabilitiesTest.js

```diff
@@ -14,7 +14,7 @@ describe('Main: Currently testing getConfig, getVersion and getCapabilities', ()
   } = require('./helpers/pactHelper.js')
 
   it('checking method : getConfig', async () => {
-    const provider = createProvider(false, true)
+    const provider = createProvider(false, false)
```

Comment:
```suggestion
    const provider = createProvider()
```
as `false` is the default we could get rid of the parameters where-ever its false for both